README.md

```diff
@@ -27,6 +27,36 @@ Installers
 Refer to the [downloads page][etcher] for the latest pre-made
 installers for all supported operating systems.
 
+#### Debian and Ubuntu based Package Repository (GNU/Linux x64/x86)
+
+1. Save the following as `/etc/apt/sources.list.d/etcher.list`:
+
+    ```
+    deb https://dl.bintray.com/resin-io/debian stable etcher
+    ```
+
+2. Trust Bintray.com's GPG key:
+
+    ```sh
+    sudo apt-key adv --keyserver pgp.mit.edu --recv-keys 379CE192D401AB61
+    ```
+
+3. Update and install:
+
+    ```sh
+    sudo apt-get update
+    sudo apt-get install etcher-electron
+    ```
+
+#### Brew Cask (macOS)
+
+Notice the Etcher Cask has to be updated manually to point to new versions, so
+it might not refer to the latest version right after an Etcher release.
```

Comment:
...and change "right after" to "immediately after" :)
And perhaps "an Etcher release" to "a new Etcher release"